home/clients.py

```diff
@@ -12,6 +12,7 @@
                                   UpdateView)
 from django.views.generic.edit import UpdateView
 from django.shortcuts import get_list_or_404, get_object_or_404
+from django.urls import reverse_lazy
 
 
 class ClientSignUpView(CreateView):
```

Comment:
you have not formated your code

Reply:
Hoyaa this thing is optional.  And my editor formats it every time I save.  Maybe I should adjust the settings for formatting